src/compiler/commandLineParser.ts

```diff
@@ -1347,11 +1339,11 @@ namespace ts {
                 const description = descriptionColumn[i];
                 result.push(optionName && `${tab}${tab}${optionName}${ description && (makePadding(marginLength - optionName.length + 2) + description)}`);
             }
-            if (configurations.files && configurations.files.length) {
+            if (fileNames.length) {

```

Comment:
This function is `@internal` and only called in one place, so if this were nullable I think we would know.